src/components/Viewer/InformationPanel/InformationPanel.tsx

```diff
@@ -6,14 +6,20 @@ import {
   Wrapper,
 } from "src/components/Viewer/InformationPanel/InformationPanel.styled";
 import React, { useEffect, useState } from "react";
-import { ViewerContextStore, useViewerState } from "src/context/viewer-context";
+import {
+  ViewerContextStore,
+  useViewerDispatch,
+  useViewerState,
+} from "src/context/viewer-context";
 
 import AnnotationPage from "src/components/Viewer/InformationPanel/Annotation/Page";
 import { AnnotationResources } from "src/types/annotations";
 import Information from "src/components/Viewer/InformationPanel/About/About";
 import { InternationalString } from "@iiif/presentation-3";
 import { Label } from "src/components/Primitives";
 
+const UserScrollTimeout = 1500; // 1500ms without a user-generated scroll event reverts to auto-scrolling
```

Comment:
We can iterate on this further but this value would be nice to add to the configOptions.